packages/world/ts/deduplicateAbi.ts

```diff
@@ -0,0 +1,24 @@
+import { Abi, AbiItem } from "viem";
+
+function hasNamedInputs(entry: AbiItem) {
+  if (!("inputs" in entry) || !entry.inputs || !entry.inputs.length) {
+    return false;
+  }
+  return entry.inputs.some((input) => input.name);
```

Comment:
I am not sure if there is any downside to have both, named and unnamed inputs. I am favoring named inputs so I can display more useful information to users in the explorer. And I am also deduplicating same functions just because it might be confusing to have two equivalent `batchCall` shown in the UI.

As for counting, I am not sure how helpful that would be in this case because the world ABI that gets fetched dynamically does not have any named inputs while the static (imported) ABI has all of its inputs named. So it's always either 0 named inputs, or some number > 0.